src/redux/slices/tasksSlice.js

```diff
@@ -0,0 +1,40 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const taskSlice = createSlice({
+  name: "tasks",
+  initialState: [],
+  reducers: {
+    addTaskRequest: () => {},
+    addTaskSuccess: (state, action) => {
+      return [...state, action.payload];
+    },
```

Comment:
Request functions should not be empty. They are supposed to set the loading state. In the case of API calls, you’ll also have a Fail function that’ll set the error state. Also, handle the loading state in the front end by using a loader.